src/app/components/planner-list/planner-list.component.less

```diff
@@ -139,3 +203,100 @@
     align-items: inherit;
   }
 }
+
+button.btn.btn-link.dropdown-toggle:focus {
```

Comment:
this css will get applied to every button with dropdown-toggle. If we need it then add it in common file